app/src/test/java/com/keylesspalace/tusky/db/TimelineDaoTest.kt

```diff
@@ -448,7 +448,7 @@ class TimelineDaoTest {
             favourited = !even,
             bookmarked = false,
             sensitive = even,
-            spoilerText = "spoier$statusId",
+            spoilerText = "spoiler$statusId",
```

Comment:
??

Reply:
its just some string to fill something into the content warning ("spoiler")